config/sets/symfony/symfony61.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\Name\RenameClassRector;
+use Rector\Symfony\Rector\Class_\CommandPropertyToAttributeRector;
+
+# https://github.com/symfony/symfony/blob/6.1/UPGRADE-6.1.md
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->rule(CommandPropertyToAttributeRector::class);
+
+    $rectorConfig->ruleWithConfiguration(
+        RenameClassRector::class,
+        [
+            // @see https://github.com/symfony/symfony/pull/43982
+            'Symfony\Component\Serializer\Normalizer\ContextAwareDecoderInterface' => 'Symfony\Component\Serializer\Normalizer\DecoderInterface',
```

Comment:
Looking at the [UPGRADE-6.1.MD](https://github.com/symfony/symfony/blob/6.1/UPGRADE-6.1.md) I don't see the `ContextAwareDecoderInterface` and `ContextAwareEncoderInterface` being deprecated.

Those are mentioned in the PR you are referring to, but have been reversed in [PR #47150](https://github.com/symfony/symfony/pull/47150)

Reply:
Also good find thanks!